docs/use/dapp-staking/for-stakers/index.md

```diff
@@ -0,0 +1,109 @@
+| Blocks Per Era | 1800 (~6hours) | TBD | TBD |
+
+
+### Basic Staking Rewards:
+
+dApp Staking rewards are generated only if tokens have been staked on a dApp for at least 1 full `Era` and distributed on the following `Era` (Era+1);
+
+:::tip
+
+For stakers, if you stake today, you will start earning tomorrow (next era), and the rewards will be distributed the day after tomorrow.
+
+For dApps If you get into a tier today, you will start earning rewards tomorrow (next era) and the rewards will be distributed the day after tomorrow.
+
+:::
+
+For stakers, the APR is the same for all stakers, regardless of which dApp they stake on;
+For dApps, staking rewards depends on the Tier system, dApps go into rewards tier based on the amount of tokens staked by users.
+
+Rewards must be claimed before voting for a new dApp or during the new **Voting Period**.
+*(If using the Astar Portal, the interface will automatically claim the rewards when voting)*
+
+If a project is **unregistered** from dApp Staking, staked tokens on that project will still receive **basic rewards** as long as the tokens remain staked.
+
+:::warning
+
+ There is no rewards during the Unbonding period when unstaking from dApp Staking;
+
+ :::
+
+### Bonus Staking Rewards:
+
+If an user stake on dApp(s) during the **Voting Period** and keep the same staked amount or higher on a dApp through the whole **Build Period**, he is eligible for a the **Bonus Reward** for this dApp.
+
+:::info
+
+**Example**
+
+- User stake 1500 ASTR on **dApp A** and 1000 ASTR on **dApp B** during the `Voting` Period.
+- During the `Build&Earn` Period, user move 500 ASTR from **dApp B** to **dApp A**;
+- He now has 2000 ASTR on **dApp A** and 5000 ASTR on **dApp B**, so he's still eligible for the `Bonus reward` for dApp A but no longer for **dApp B** because the tokens staked on **dApp** B are less than those staked at the end of the `Voting period` (500 < 1000 ASTR).
+- To be eligible for the `Bonus reward` for **dApp B**, the user must stake an additional 500 ASTR on **dApp B** to reach 1000 ASTR.
+
+:::
+
+Bonus rewards can be claimed during the following `Voting Period`. 
+
+### Other:
+
+- The **minimum** amount to stake for users is **50 SDN or 500 ASTR.**
+- You need to keep a minimum of x tokens as transferable after staking.
+- **You need to claim to receive your rewards, we recommend claiming your staking rewards once a week.**
+- There is a unbonding period of around 5 days on Shiden and 10 days on Astar. Please note that this is based on a perfect block production of 12s. In case of any delay, your unbonding period can be a little longer.
+- The staking rewards are shown in APY by default. APY option is available for stakers who are compounding their rewards by re-staking after claiming. On the other hand, APR option is available for stakers who don’t do compounding. The difference is that APR represents the annual rate simply for earning your stake but APY takes compounding into account.
```

Comment:
Please check this with the portal team, I'm not sure they will support re-stake on launch.

Reply:
This was not a reference to the auto-compouding feature but an explanation of the difference between APR and APY. I added "manually" to the sentence to make it more understandable that the user has to re-stake his rewards himself to get the APY.